locallib.php

```diff
@@ -477,19 +477,40 @@ public function get_form_elements_for_user($submission, MoodleQuickForm $mform,
                 );
 
                 foreach ($views['data'] as $view) {
-                    $viewurl = "/view/view.php?id=" . $view['id'];
-                    $anchor = $this->get_preview_url($view['title'], $viewurl, strip_tags($view['description']));
-                    $mform->addElement('radio', 'viewid', '', $anchor, 'v' . $view['id']);
+                    if ($view['submissionoriginal'] == 0) {
```

Comment:
What does `$view['submissionoriginal'] == 0` mean? Maybe add a comment explaining it? Same below on line 500